chain/src/test/java/org/ethereum/beacon/chain/StreamTests.java

```diff
@@ -43,4 +42,25 @@ public void test1() throws InterruptedException {
 
     Thread.sleep(1000L);
   }
+
+  @Test
+  @Ignore
+  public void intervalTest1() throws InterruptedException {
```

Comment:
What are you testing here?

Reply:
Just some experiments. Actually `SlotTicker` should consist just of several lines, but `Flux.interval` has missing feature: it accounts `initialDelay` only on subscribing. Will be working on this